content/docs/self-hosted-runners.md

```diff
@@ -391,6 +391,51 @@ provisioned through environment variables instead of files.
 </tab>
 </toggle>
 
+#### Cloud Compute Resource Manual Cleanup
+
+In very rare cases, you may need to cleanup CML cloud resources manually.
+An example of such a problem can be seen
+[when an EC2 instance ran out of storage space](https://github.com/iterative/cml/issues/1006).
+
+The following sections contain lists of all the resources you may need to
+manually cleanup in the case of a failure.
+
+<toggle>
+<tab title="AWS">
```

Comment:
instance / key-pair are not specific to AWS

Reply:
Are you saying I should remove the tabbing or are you just talking about the key pair stuff?

Reply:
I would remove the toggle/tab and generally refer to "instances" vs "EC2"